Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2537 -- Remove Resources page

Resolves [EREGCSC-2537](https://jiraent.cms.gov/browse/EREGCSC-2537)

**Description**

The Resources page was a beautiful mess.  We learned a lot while making it, and it had a good run, but now it's time to put it to rest.

**This pull request changes:**

- Redirects `/resources/` route to `/subjects/` page
- Removes all code, styles, components, and dependencies that are only used in the Resources page

**Steps to manually verify this change:**

1. Visit [experimental deployment](https://8ju858oaxa.execute-api.us-east-1.amazonaws.com/dev1190)
2. Try to visit `/resources/` and make sure you end up at `/subjects/`
3. Browse around the page and validate that there are no side effects.  Things should look, feel, and work as expected.
